mllm/backends/ascend/memory/MemoryBlock.hpp

```
@@ -0,0 +1,17 @@
+// Copyright (c) MLLM Team.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <cstdint>
+#include <cstddef>
+
+namespace mllm::ascend {
+
+struct MemoryBlock {
```

Clang (14.0.6)

[error] constructor does not initialize these fields: block_id_, block_size_

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)